src/dswx_sar/mosaic_gcov_frame.py

```diff
@@ -140,17 +134,15 @@ def write_rtc_geotiff(
         Returns
         -------
         layover_exist: bool
-            Boolean which indicates if a layoverShadowMask layer exisits in input RTC
+            Boolean which indicates if a layoverShadowMask layer exisits
+            in input RTC
         geogrid_in: DSWXGeogrid object
-            A dataclass object  representing the geographical grid configuration
-            for an RTC (Radar Terrain Correction) run.
+            A dataclass object  representing the geographical grid
+            configuration for an RTC (Radar Terrain Correction) run.
         """
 
         # Reproject geotiff
         most_freq_epsg = majority_element(epsg_array)
-
-        geogrid_in = DSWXGeogrid()
-
         designated_value = np.float32(500)
```

Comment:
I hardcoded the replacement value here. It could be a configurable item in the YAML file next round.